hw/ip/entropy_src/doc/entropy_src_dv_plan.md

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
Apologies for this. This file needs to move to 
doc/dv_plan/index.md

I will make a fix in the uvmdvgen script.

Reply:
Actually nevermind. I will take care of it.